src/functionalTest/resources/scenarios/RIA-3390-request-home-office-data-success.json

```diff
@@ -0,0 +1,32 @@
+{
+  "description": "RIA-3390 Request Home Office data and gets data successfully",
+  "enabled": "{$featureFlag.isHomeOfficeIntegrationEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "CaseOfficer",
+    "input": {
+      "eventId": "requestHomeOfficeData",
+      "state": "appealSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "homeOfficeReferenceNumber": "1212-0099-0062-8083"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "homeOfficeReferenceNumber": "1212-0099-0062-8083",
+        "isHomeOfficeIntegrationEnabled": "Yes",
+        "contactPreferenceDescription": "Text message",
+        "appealTypeDescription": "Refusal of protection claim",
+        "homeOfficeSearchStatus": "SUCCESS"
+      }
+    }
+  }
+}
```

Comment:
@divisathyan we need to cover this scenario as well **error message when event has already retrieved the data successfully***

Reply:
Added functional test